Rehlate Landing Page/index.html

```diff
@@ -0,0 +1,644 @@
+                        <rect width="22" height="22" fill="white" transform="translate(40 18) rotate(90)"/>
+                        </clipPath>
+                        </defs>
+                        </svg>
+                        
+                </div>
+                
+            </div>
+            
+        </div>
+    </section >
+    <section id="Contact" class="contact">
+        <div class="container">
+            <div class="text">
+                <p class="GrayColor">هل لديك سؤال؟</p>
+                <h2 class="darkBlack">يسعدنا التواصل معك</h2>
+            </div>
+            <div class="d-flex column gap-25">
+                <div class="form-contact RiseNShine300-bg">
+                    <form action="index.html">
+                      <label for="name"><h3>الاسم</h3></label>  
+                      <input type="text" name="name" id="name" placeholder="يرجى تسجيل اسمك الكامل">
+                      <label for="email"><h3>البريد الإلكتروني</h3></label>
+                      <input type="email" name="email" id="email" placeholder="سجل بريدك الشخصي"/>
+                      <label for="subject"><h3>الموضوع</h3></label>
+                      <textarea name="subject" placeholder="ما هو الاستفسار او المشكلة التي لديك" id="subject">
+                        
+                      </textarea>
+                      <div class="button ">
+                        <button class="darkBlack-bg"><a class="primaryColor" href="#rehalate">ارسل رسالة</a></button>
+                      </div>
+                    </form>
+                </div>
+                <div class="flex-column contact-help">
+                    <div class=" d-flex column  gap-25">
+                        <div class="contact-item-help">
+                            <div class="contact-item-img center ">
+                                <img src="./assets/images/help.png" alt="help-photo"/>
+                            </div>
+                        </div>
+                            <div class="contact-item-content">
+                                <p>نقدم لك المساعدة</p>
+                                <p>إن كان لديك اي مشكلة في البلد الذي تسافر 
+                                    إليه سنقوم بمساعدك في هذا الموضوع.</p>
+                            </div>
+                            
+                    </div>
+                    <div class=" d-flex column gap-25">
+                        <div class="contact-item-help">
+                            <div class="contact-item-img center">
+                                <img src="./assets/images/communication.png" alt="help-photo"/>
+                            </div>
+                        </div>
+                            <div class="contact-item-content">
+                                <p>للتواصل</p>
+                                <p>يمكنك التواصل معنا عبر الفورم أو عبر البريد
+                                    الإلكتروني: company@mail.com</p>
```

Comment:
Please ensure that this email color is `#755AE8`

Reply:
done .